templates/sag-mft-config/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+
+# if managed image
+if [ -d $SAG_HOME/profiles/SPM ] ; then
+    # point to local SPM
+	export portName= FTP
+    export CC_SERVER=http://localhost:8092/spm
+    export is_instance_name=${is_instance_name:-default}
+	
+	
+    echo "Verifying managed container $CC_SERVER ..."
+    sagcc get inventory products -e MFTSupport --wait-for-cc
+
+    export CC_WAIT=180
+    
+    echo "Verifying fixes ..."
+    sagcc get inventory fixes -e MAT_10.1_Server_Fix
```

Comment:
Since this template does not install fixes, you should not assert fix present in this test.sh